cabal.project

```diff
@@ -58,6 +58,10 @@ optimization: False
 constraints: crypton < 0, crypton-connection < 0, crypton-x509 < 0, crypton-x509-store < 0, crypton-x509-system < 0, crypton-x509-validation < 0
 constraints: warp < 3.3.26
 
+-- wreq-0.5.4.1 doesn't seem to work with ghc-8.6.5
+if (impl(ghc <= 8.6))
```

Comment:
Upper bounds should never be set with `<=` since that is excluding minor versions (which normally would only contain fixes). So as I read this, 8.6.5 is not matching this predicate.

Better use `< 8.8`.

Reply:
Ah didn't realise that, hopefully this will get CI to work for 8.6.5 now.